mods/voidmulti.sc2mod/base.sc2data/gamedata/abildata.xml

```diff
@@ -629,6 +710,52 @@
         <CmdButtonArray index="Execute">
             <Flags index="ShowInGlossary" value="0"/>
         </CmdButtonArray>
+        <InfoArray index="0">
+            <SectionArray index="Abils">
+                <DurationArray index="Delay" value="0"/>
+            </SectionArray>
+            <SectionArray index="Actor">
+                <DurationArray index="Delay" value="0"/>
+            </SectionArray>
+            <SectionArray index="Mover">
+                <DurationArray index="Delay" value="0"/>
+                <DurationArray index="Duration" value="0"/>
+            </SectionArray>
+            <SectionArray index="Stats">
+                <DurationArray index="Delay" value="0"/>
```

Comment:
**undocumented**:
- Swarm Host's Unburrow stats transformation delay was reduced from 0.5 to 0
- Swarm Host's Unburrow actor transformation delay was reduced from 0.5 to 0
- Swarm Host's Unburrow abilities transformation delay was reduced from 0.5 to 0
- Swarm Host's Unburrow mover transformation delay was reduced from 0.5 to 0
- Swarm Host's Unburrow mover transformation duration was reduced from 1 to 0

This allows orders being queued after giving the unburrow order.

The final duration is random delay + highest duration.